src/controller.rs

```diff
@@ -728,16 +728,20 @@ impl AppInstanceLike {
             metadata: metadata.clone(),
             rules: Some(vec![PolicyRule {
                 api_groups: Some(
-                    ["apiextensions.k8s.io"]
+                    ["apiextensions.k8s.io", "rbac.authorization.k8s.io"]
                         .iter()
                         .map(|s| s.to_string())
                         .collect(),
                 ),
                 resources: Some(
-                    ["customresourcedefinitions"]
-                        .iter()
-                        .map(|s| s.to_string())
-                        .collect(),
+                    [
+                        "customresourcedefinitions",
+                        "clusterrolebindings",
+                        "clusterroles",
```

Comment:
Adding extra roles here does fix the issue; however, it does not seem very elegant.

Then again, this is also the `setup_cluster_roles` function so it is rather apt for what this is doing :thinking: